templates/log4j.properties.j2

```diff
@@ -0,0 +1,58 @@
+# Define some default values that can be overridden by system properties
+zookeeper.root.logger=INFO, CONSOLE, ROLLINGFILE
+zookeeper.console.threshold=INFO
+zookeeper.log.dir=.
+zookeeper.log.file=zookeeper.log
+zookeeper.log.threshold=DEBUG
+zookeeper.tracelog.dir=.
```

Comment:
this is not the app log dir, it is the data log dir (i.e. used by zk to store uncommitted / replicated changes) the log file location is controlled by using the ZOO_LOG_DIR env variable which is used in the various scripts under its /bin directory